cmd/sanssh/main.go

```diff
@@ -46,6 +46,7 @@ import (
 	_ "github.com/Snowflake-Labs/sansshell/services/healthcheck/client"
 	_ "github.com/Snowflake-Labs/sansshell/services/localfile/client"
 	_ "github.com/Snowflake-Labs/sansshell/services/packages/client"
+	_ "github.com/Snowflake-Labs/sansshell/services/power/client"
```

Comment:
it's not too important, but good idea to also add this to cmd/sansshell-server/main.go

Reply:
ack